init-sensor.h

```diff
@@ -7,8 +7,9 @@ struct SensorConfig {
   uint8_t sensorType;
 };
 
-const char *sensorFileName = "/sensorConfig.json";
+const char *sensorFileName = "/CurrentSensorConfig.json";
```

Comment:
what does flashing means here? how do you chnage sensorType property? it can be hardcoded like you hardcode all of other configs, or sent from phone - in this case sensorType should be part of qr. what qr do you use? because I cannot see sensorType encoded in your qr?

or how do you reproduce this bug?